pep-0567.rst

```diff
@@ -551,6 +604,53 @@ code unmodified, but will automatically enable support for
 asynchronous code.
 
 
+Examples
+========
+
+Converting code that uses threading.local()
+-------------------------------------------
+
+A typical code that uses ``threading.local()`` usually looks like
+the following snippet::
+
+    class mylocal(threading.local):
+        # Subclass threading.local to specify a default value.
+        value = 'spam'
+
+    mylocal = mylocal()
```

Comment:
This has a variable of the same name as its type. Please be more creative. :-)

(In general, the example would probably benefit from being specific to some application rather than being generic like this. You don't have to add more code to make the example complete, just pick variable names that suggest a use case -- perhaps "precision" would do?)

Reply:
fixed